app/controllers/employees_controller.rb

```diff
@@ -56,6 +63,10 @@ def employee_params
     params.require(:employee).permit(:slack_username, :started_on, :time_zone)
   end
 
+  def existing_employee(slack_username)
+    !!Employee.where(slack_username: slack_username)
```

Comment:
I removed this method entirely. You were right when you said that that method already existed.